### PR TITLE
[WIP] Use ObjectMapper from JacksonUtil

### PR DESCRIPTION
**Pull request by van-vanich**
## Pull Request description

Issue for PR: https://github.com/thingsboard/thingsboard/issues/6046

Use in all project methods for ObjectMapper from JacksonUtil. Also registered in mapper JavaTimeModule to parse Instant objects, because Java 8 date/time type java.time.Instant is not supported by default.

## General checklist

- [x] This PR has the corresponding CE PR or contains unique code changes related to PE only (does not require same changes in CE).
- [x] You have reviewed the guidelines [document](https://docs.google.com/document/d/1wqcOafLx5hth8SAg4dqV_LV3un3m5WYR8RdTJ4MbbUM/edit?usp=sharing).
- [x] PR name contains fix version. For example, "[3.3.4] Hotfix of some UI component" or "[3.4] New Super Feature".
- [x] Description references specific [issue](https://github.com/thingsboard/thingsboard/issues).
- [x] Description contains human-readable scope of changes.
- [x] Description contains brief notes about what needs to be added to the documentation.
- [x] No merge conflicts, commented blocks of code, code formatting issues.
- [x] Changes are backward compatible or upgrade script is provided.

## Back-End feature checklist

- [x] Added corresponding unit and/or integration test(s). Provide written explanation in the PR description if you have failed to add tests.
- [x] If new dependency was added: the dependency tree is checked for conflicts.
- [x] If new service was added: the service is marked with corresponding @TbCoreComponent, @TbRuleEngineComponent, @TbTransportComponent, etc.
- [x] If new REST API was added: the RestClient.java was updated, issue for [Python REST client](https://github.com/thingsboard/thingsboard-python-rest-client) is created.

### CODE REVIEW

File: application/src/main/resources/thingsboard.yml [hunk 1]
```diff
@@ -519,7 +519,7 @@ spring:
     database-platform: "${SPRING_JPA_DATABASE_PLATFORM:org.hibernate.dialect.PostgreSQLDialect}"
   datasource:
     driverClassName: "${SPRING_DRIVER_CLASS_NAME:org.postgresql.Driver}"
-    url: "${SPRING_DATASOURCE_URL:jdbc:postgresql://localhost:5432/thingsboard}"
+    url: "${SPRING_DATASOURCE_URL:jdbc:postgresql://localhost:5433/thingsboard}"
```
Comment (dmytro-landiak) [inline application/src/main/resources/thingsboard.yml:522]: pls change this back

Comment (dmytro-landiak) [reply application/src/main/resources/thingsboard.yml:522]: otherwise, LGTM

Comment (van-vanich) [reply application/src/main/resources/thingsboard.yml]: > pls change this back

Sorry, I fixed this unnecessary

**Comment by dmytro-landiak** [comment]
WIP - need to fix errors.

File: application/src/main/java/org/thingsboard/server/actors/ActorSystemContext.java [hunk 2]
```diff
@@ -108,7 +109,7 @@
 @Component
 public class ActorSystemContext {
 
-    protected final ObjectMapper mapper = new ObjectMapper();
+    protected final ObjectMapper mapper = JacksonUtil.OBJECT_MAPPER;
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/actors/ActorSystemContext.java:112]: This is wrong. You should remove this variable and use static methods from JacksonUtil where the "mapper" was used. Same for all other files.

**Comment by van-vanich** [comment]
@ashvayka done

File: application/src/main/java/org/thingsboard/server/actors/ActorSystemContext.java [hunk 3]
```diff
@@ -478,7 +475,7 @@ private String toString(Throwable e) {
     }
 
     private JsonNode toBodyJson(String serviceId, ComponentLifecycleEvent event, Optional<Exception> e) {
-        ObjectNode node = mapper.createObjectNode().put("server", serviceId).put("event", event.name());
+        ObjectNode node = JacksonUtil.getObjectMapper().createObjectNode().put("server", serviceId).put("event", event.name());
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/actors/ActorSystemContext.java:478]: JacksonUtil.newObjectNode()

File: application/src/main/java/org/thingsboard/server/actors/ActorSystemContext.java [hunk 4]
```diff
@@ -532,9 +529,9 @@ private void persistDebugAsync(TenantId tenantId, EntityId entityId, String type
                 event.setEntityId(entityId);
                 event.setType(DataConstants.DEBUG_RULE_NODE);
 
-                String metadata = mapper.writeValueAsString(tbMsg.getMetaData().getData());
+                String metadata = JacksonUtil.getObjectMapper().writeValueAsString(tbMsg.getMetaData().getData());
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/actors/ActorSystemContext.java:532]: JacksonUtil.toString

**Review by dmytro-landiak — APPROVED** [review]

File: application/src/main/java/org/thingsboard/server/controller/BaseController.java [hunk 5]
```diff
@@ -828,19 +825,14 @@ public static Exception toException(Throwable error) {
     }
 
     protected <E extends HasName> String entityToStr(E entity) {
-        try {
-            return json.writeValueAsString(json.valueToTree(entity));
-        } catch (JsonProcessingException e) {
-            log.warn("[{}] Failed to convert entity to string!", entity, e);
-        }
-        return null;
+        return JacksonUtil.toString(JacksonUtil.valueToTree(entity));
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/controller/BaseController.java:828]: You are affecting the behavior of the method. Before your change, this method never throws exceptions. After your change, you may receive the unchecked IllegalArgumentException. Please make sure you do not change the behavior of the code during refactoring and review all changes according to this comment.

File: application/src/main/java/org/thingsboard/server/actors/device/DeviceActorMessageProcessor.java [hunk 6]
```diff
@@ -817,7 +817,7 @@ private void saveRpcRequestToEdgeQueue(ToDeviceRpcRequest msg, Integer requestId
         edgeEvent.setEntityId(deviceId.getId());
         edgeEvent.setType(EdgeEventType.DEVICE);
 
-        ObjectNode body = mapper.createObjectNode();
```
Comment (YevhenBondarenko) [inline application/src/main/java/org/thingsboard/server/actors/device/DeviceActorMessageProcessor.java:820]: plz remove mapper and import

**Pull request closed without merging**